store/imagestore/store.go

```diff
@@ -426,7 +426,7 @@ func (s *Store) ReadStream(key string) (io.ReadCloser, error) {
 // (i.e. the hash of the uncompressed ACI)
 // latest defines if the aci has to be marked as the latest. For example an ACI
 // discovered without asking for a specific version (latest pattern).
-func (s *Store) WriteACI(r io.ReadSeeker, fetchInfo ACIFetchInfo) (string, error) {
+func (s *Store) WriteACI(r io.ReadSeeker, latest bool) (string, error) {
 	// We need to allow the store's setgid bits (if any) to propagate, so
```

Comment:
In general, I think it may make sense to keep ACIFetchInfo as a single field struct, but do we currently plan to collect any other fetching info here?